File: CS/CustomReportLabelAndDeisgner/CustomXRLblDesigner.cs
using System;
using System.Collections.Generic;
using DevExpress.XtraReports.Design;
using System.ComponentModel.Design;

namespace CustomReportControl {
    internal class CustomXRLblDesigner : XRLabelDesigner {
        public CustomXRLblDesigner()
            : base() {
        }
        protected override void RegisterActionLists(DesignerActionListCollection list) {
            list.Add(new XRLabelDesignerCustomActionList(this));
        }
    }
}

File: VB/CustomReportLabelAndDeisgner/CustomXRLblDesigner.vb
Imports System
Imports System.Collections.Generic
Imports DevExpress.XtraReports.Design
Imports System.ComponentModel.Design

Namespace CustomReportControl
    Friend Class CustomXRLblDesigner
        Inherits XRLabelDesigner

        Public Sub New()
            MyBase.New()
        End Sub
        Protected Overrides Sub RegisterActionLists(ByVal list As DesignerActionListCollection)
            list.Add(New XRLabelDesignerCustomActionList(Me))
        End Sub
    End Class
End Namespace
